Cargo.lock

```diff
@@ -9106,7 +9152,7 @@ checksum = "b3c129550b3e6de3fd0ba67ba5c81818f9805e58b8d7fee80a3a59d2c9fc601a"
 dependencies = [
  "proc-macro2",
  "quote",
- "syn 2.0.32",
+ "syn 2.0.68",
```

Comment:
Lets have these package upgrades in a separate PR so that we can bisect this out easier in case something goes awfully wrong.